infrastructure/website.template

```diff
@@ -1,196 +1,225 @@
-                                            "origin-access-identity/cloudfront/",
-                                            {
-                                                "Ref": "DevOriginAccessIdentity"
-                                            }
-                                        ]
-                                    ]
-                                }
-                            }
-                        }
-                    ],
-                    "PriceClass": "PriceClass_100",
-                    "ViewerCertificate": {
-                        "AcmCertificateArn": {
-                            "Ref": "PassrCertificate"
-                        },
-                        "MinimumProtocolVersion": "TLSv1.2_2018",
-                        "SslSupportMethod": "sni-only"
-                    }
-                },
-                "Tags": [
-                    {
-                        "Key": "project",
-                        "Value": "Passr"
-                    },
-                    {
-                        "Key": "env",
-                        "Value": "dev"
-                    }
-                ]
-            }
-        },
-        "DevCloudfrontRoute53Record": {
-            "Type": "AWS::Route53::RecordSet",
-            "Properties": {
-                "ResourceRecords": [
-                    {
-                        "Fn::GetAtt": [
-                            "DevCloudfront",
-                            "DomainName"
-                        ]
-                    }
-                ],
-                "HostedZoneId": {
-                    "Ref": "PassrCAHostedZone"
-                },
-                "Name": "dev.passr.ca",
-                "Type": "CNAME",
-                "TTL": "300"
-            }
-        }
-    },
-    "Outputs": {
-        "DevOriginAccessIdentity": {
-            "Description": "Dev Cloudfront Origin Access Identity",
-            "Export": {
-                "Name": "dev-cloudfront-origin-access-identity"
-            },
-            "Value": {
-                "Ref": "DevOriginAccessIdentity"
-            }
-        }
-    }
+	"AWSTemplateFormatVersion": "2010-09-09",
+	"Description": "Template for Website Deployment",
+	"Metadata": {},
+	"Parameters": {
+		"DevOriginId": {
+			"Type": "String",
+			"Default": "passr-dev-origin-access-identity",
+			"Description": "Origin access identity ID for passr dev bucket"
+		}
+	},
+	"Mappings": {},
+	"Conditions": {
+		"CreateStack": {
+			"Fn::Equals": [
+				{
+					"Ref": "AWS::Region"
+				},
+				"us-east-1"
+			]
+		}
+	},
+	"Resources": {
+		"PassrCAHostedZone": {
+			"Type": "AWS::Route53::HostedZone",
+			"Condition": "CreateStack",
+			"Properties": {
+				"Name": "passr.ca"
+			}
+		},
+		"PassrCertificate": {
+			"Type": "AWS::CertificateManager::Certificate",
+			"Condition": "CreateStack",
+			"Properties": {
+				"DomainName": "passr.ca",
+				"SubjectAlternativeNames": [
+					"*.passr.ca"
+				],
```

Comment:
Is this saying anything may be put before .passr.ca and it'll still send them to passr.ca?

Reply:
So this is actually a certificate for our domain, the `*` includes all subdomains (dev.passr.ca etc). This is a way of getting HTTPS on our site. SSL Certificates are a security feature that allow you to be sure that the owner of a domain is in fact the one that you are connecting to. Over non https connections, websites are vulnerable to multiple attacks, like man in the middle, spoofing and more. 

You can read more about certificates here: https://www.globalsign.com/en/ssl-information-center/what-is-an-ssl-certificate